### DISCUSSION TITLE
Is there a procedure for renaming a repository?

### DISCUSSION THREAD

**Discussion by axi92** [SwiftPackageIndex/SwiftPackageIndex-Server#3345] [Q&A] [answered]
Should I just rename the repo and make a PR that fixes the link?

**Comment by daveverwer** [comment, marked as answer]
We handle most renames automatically, so there's usually no need to do anything. If you use GitHub's rename feature, they will redirect your old package URL to the new URL and our nightly process that runs will notice that and will raise a PR that removes the old package and adds the new one.

The only exception is if you didn't use the GitHub rename function, and manually made a new repo and deleted the old one. In that case, we will pick up and delete the old repo, but we won't auto-discover the new one.
